src/packages/solid/components/Icon/Icon.tsx

```diff
@@ -40,8 +40,8 @@ const Icon: Component<IconProps> = (props: IconProps) => {
       {...props}
       // TODO, wait to see if fixed is needed
       src={props.icon}
-      y={10}
-      x={10}
+      // y={10}
```

Comment:
had to comment these out or the x & y in Checkbox wouldn't update ( or I was doing it wrong)

Reply:
Yes - if you need to be able to change the x + y it can't be set on the component, otherwise the props won't be applied.